lib/disposable.dart

```diff
@@ -181,4 +193,10 @@ abstract class Disposable implements _Disposable {
     _didDispose.complete();
     return null;
   }
+
+  void _throwIfNull(dynamic subscription, String name) {
```

Comment:
I think ArgumentError.notNull is the majority of what a utility could be anyway. I haven't run into much use for a re usable error throwing helper.